atmo/templates/atmo/spark-job-results.html

```diff
@@ -0,0 +1,39 @@
+{% extends "atmo/base.html" %}
+
+{% block head_title %}Spark job results {{ job.identifier }}{% endblock %}
+
+{% block content %}
+<div class="page-header">
+    <h2>Spark Job results <small>{{ job }}</small></h2>
+</div>
+
+<h3>Job Artifacts</h3>
```

Comment:
No need to prefix the header with "Job" here, we know we're on the Job detail page. Also you brought me to take another look at the panels, I think they'd work nicely here: http://getbootstrap.com/components/#panels-list-group with some explanation in the panel body (e.g. how to use the private URL with AWS CLI) instead of bare list groups.